src/excalidraw-app/collab/types.ts

```diff
@@ -0,0 +1,5 @@
+export enum UserIdleState {
+  Active = "active",
+  Idle = "idle",
+  Away = "away",
```

Comment:
```suggestion
  ACTIVE = "active",
  AWAY = "away",
  IDLE = "idle",
```

Maybe all capital? (and ordered)

Reply:
Done!